src/js/components/MaskedInput/maskedinput.stories.js

```diff
@@ -0,0 +1,205 @@
+import React, { Component } from 'react';
+import { storiesOf } from '@storybook/react';
+import { Box, Grommet, MaskedInput } from 'grommet';
+import { grommet } from 'grommet/themes';
+
+class TimeMaskedInput extends Component {
+  state = { value: '' };
+
+  onChange = event => {
+    this.setState({ value: event.target.value });
+  };
+
+  render() {
+    const { value } = this.state;
+    return (
+      <Grommet full theme={grommet}>
+        <Box fill align="center" justify="center">
+          <Box width="medium">
+            <MaskedInput
+              mask={[
+                {
+                  length: [1, 2],
```

Comment:
just thinking out loud, the `length` is singular but can receive an array. I wonder if we should always make an array and call it `lengths` ? similar to `options`

Reply:
I think that's making things a bit harder for the caller.

Reply:
agreed